ext/mysql2/result.h

```diff
@@ -1,5 +1,6 @@
 #ifndef MYSQL2_RESULT_H
 #define MYSQL2_RESULT_H
+#include <stdbool.h>
```

Comment:
According to [Bug #85131	Remove my_bool](https://bugs.mysql.com/bug.php?id=85131)

> - mysql.h now requires a C++ or C99 header to compile. (All supported platforms have C99 compilers, but some are C89 by default and need mode switches like --std=gnu99 or --std=c99.)

Then as long as it is one of the "supported platforms", C99 should be supported.

I'm not familiar with Windows or Solaris C Compiler but googled and found these links:

* [What's New for Visual C++ in Visual Studio 2013
](https://msdn.microsoft.com/en-us/library/hh409293(v=vs.120).aspx)

>> Supports these ISO C99 language features:
>> _Bool


* [Sun Studio 12: C User's Guide
](http://docs.oracle.com/cd/E19205-01/819-5265/bjate/index.html)

>> B.2.73 -xc99[=o]
>> The -xc99 option controls compiler recognition of the implemented features from the C99 
>> standard (ISO/IEC 9899:1999, Programming Language - C).